### DISCUSSION TITLE
Feature: Achievements

### DISCUSSION THREAD

**Discussion by KaziPlays** [StrataSource/Portal-2-Community-Edition#759] [Ideas]
### Which component should be improved?

Other

### Describe your suggestion

I saw that this was planned and there was not an issue for it so I opened one

### Expected result

Achievements.

**Comment by KaziPlays** [comment]
Bring back the advanced maps

**Comment by vrad-exe** [comment]
Please go into more detail about what you mean by this. Mods with their own appids can already give achievements, and it's not possible to enable that system for sourcemods.

An advanced chambers menu can already be implemented through Panorama.

**Reply by KaziPlays** [reply]
This discussion was about P2:CE's achievements, not other mods's achievements.
